docs/_docs/templates.md

```diff
@@ -105,6 +105,35 @@ you come up with your own tags via plugins.
         </p>
       </td>
     </tr>
+    <tr>
+      <td>
+        <p class="name"><strong>Date to String in ordinal US style</strong></p>
+        <p>Format a date to ordinal, UK, short format.</p>
```

Comment:
Could you replace `UK` here with `US` since it looks like this is for the US format based on the context around it?